WordPress/gutenberg
# Separator: Move box shadow and outline

## What?

Moves the box shadow and outline that are present when the block is focused on.

Fixes https://github.com/WordPress/gutenberg/issues/41543 and
https://github.com/WordPress/gutenberg/issues/36730

## Why?

The shadow was not displaying correctly:
- It was too close to the block, making the selected color for the block difficult to see.
- When the separator has a custom style, like in Twenty Twenty, the box shadow did not display around the whole block.

## How?

Removes the default style and adds a box shadow and outline that is placed further away from the block.

## Testing Instructions





1. In either editor, add a few separator blocks. Please use all 3 styles: default, dots, wide, with and without color.
2. Select the block with a mouse or other pointer. Confirm that the shadow is showing and that the color of the block is visible.
3. Deselect the block and now navigate to the block using the keyboard. Confirm that the shadow is showing and that the color of the block is visible when the block is selected.


## Screenshots or screencast 
Before
![The shadow is only visible around parts of the block](https://user-images.githubusercontent.com/7422055/181697276-1c4db433-7b33-4a82-82c7-3863c33a1d1f.png)

After:
![181693904-e53f7fd0-223e-41eb-82e9-9d8adb046fe3](https://user-images.githubusercontent.com/7422055/181697891-00813a58-0099-4bcf-b95e-cc32eb9415fe.png)
